config/ext/rspec.yml

```
@@ -298,7 +298,7 @@ RSpec/ExpectActual:
 
 RSpec/ExpectChange:
   Description: Checks for consistent style of change matcher.
-  Enabled: true
+  Enabled: false
```

You might want to add the reason as comment, so the original discussion is easy to lookup.

```suggestion
  Enabled: false # We decided to disable this cop https://github.com/makandra/makandra-rubocop/issues/29
```